docs/pages/concepts/offloading/_index.md

```diff
@@ -5,42 +5,32 @@ weight: 4
+This solution enables the transparent extension of the local cluster, with the new Node (and its capabilities) seamlessly taken into account by the vanilla Kubernetes scheduler when selecting the best place for the workloads execution.
+At the same time, this approach is fully compliant with standard Kubernetes APIs, hence allowing to interact with and inspect offloaded Pods just as if they were executed locally.
 
-In Kubernetes, each physical node is managed by the [kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/), a process running in the hosting machine that interfaces with the Kubernetes API server and handles the lifecycle of the pods scheduled on it.
+### Liqo Virtual Kubelet
 
-Since the foreign cluster is represented by a virtual node in the home cluster, offloading a pod on the remote cluster corresponds to scheduling it on a specific node.
-When the [virtual kubelet](#virtual-kubelet) creates a new pod in the foreign cluster, the foreign scheduler elects one *remote, physical* node as host for the received pod, while the *remote, physical* kubelet managing that pod takes care of the containers' execution.
+In Kubernetes, each physical node is managed by the [kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/), a process running in the hosting machine that interfaces with the Kubernetes API server and handles the lifecycle of both the Node itself and the Pods therein hosted.
+Similarly, the virtual node abstraction is implemented through an extended version of the [Virtual Kubelet project](https://github.com/virtual-kubelet/virtual-kubelet#liqo-provider).
+The virtual kubelet replaces a traditional kubelet when the controlled entity is not a physical node, allowing to control arbitrary objects through standard Kubernetes APIs.
 
-According to this mechanism, the offloading of a pod to the foreign cluster is fully compliant with the Kubernetes control plane: the home cluster can control all the remote jobs by interacting with the big node that represents the remote cluster. 
+In the context of Liqo, the virtual kubelet interacts with both the local and the remote clusters, and it is in charge of three main tasks:
 
-### Virtual Kubelet
+1. Creating the (virtual) node resource and reconciling its status with respect to the negotiated configuration, as described in the [node lifecycle](/concepts/offloading/node-lifecycle) section.
+2. Offloading the local Pods scheduled on the corresponding (virtual) node to the remote cluster, while aligning their status, as detailed in the [pod offloading](/concepts/offloading/pod-offloading) section.
+3. Propagating and synchronizing the accessory artifacts required for proper execution of the offloaded workloads, a feature we call [resource reflection](/concepts/offloading/resource-reflection).
```

Comment:
It would be nice to make an example of what "accessory artifacts" are. E.g.:
==> Propagating and synchronizing the accessory artifacts (e.g., XXXX) required....